AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1253,6 +1253,7 @@ imslp.org#%#AG_onLoad(function(){var b=document.getElementById("sm_dl_wait");if(
 ||cdn.web.tv/asset/javascripts/smothScroll.min.js
 ||serialuk.net/js/copyright.js
 !--- CSS ---
+fmkorea.com#$##bd_capture { background: none!important; }
```

Comment:
Why do you need this rule?

Reply:
To remove the `div` property displaying your IP address or user ID.

Reply:
Example: https://www.fmkorea.com/best2/3726267737